src/Mono.Android/Android.Runtime/AndroidEnvironment.cs

```diff
@@ -283,7 +283,7 @@ static string GetDefaultTimeZone ()
 			}
 		}
 
-		[DllImport ("__Internal", CallingConvention = CallingConvention.Cdecl)]
+		[DllImport ("monodroid", CallingConvention = CallingConvention.Cdecl)]
```

Comment:
since we are hitting all these locations, can we change these to use a const? so we can update it if need be in one place.

Reply:
Good idea. I need to update this PR to match the recent changes on `master`